### ISSUE TITLE
Shall we pass all the system properties to Gatling?

### ISSUE THREAD

**Issue by lkishalmi** [lkishalmi/gradle-gatling-plugin#23]
Right now all system properties ape passed to Gatling execution.

**Comment by eshepelyuk** [comment]
After implementing #25 - this will become configurable

**Comment by eshepelyuk** [comment]
What are the suggestions ?
Only pass some particular properies ?

**Comment by yarulan** [comment]
If you decide not to, please make it possible to specify which ones to pass. Something like
```
gatling {
    systemProperties System.properties.subMap(["host", "port"])
}
```

**Comment by eshepelyuk** [comment]
Will be addressed in #94, please fell free to feedback there
